crates/latch_core/src/ecs/storage/column.rs

```diff
@@ -0,0 +1,112 @@
+// column.rs - Component column storage with double-buffering
+//
+// Each column stores all instances of a single component type in a
+// Structure-of-Arrays layout for cache efficiency.
+//
+// # Double-Buffering
+//
+// Each column maintains two buffers to enable deterministic parallel updates:
+// - Systems read from the "current" buffer (stable state from last tick)
+// - Systems write to the "next" buffer (new state for next tick)
+// - After all systems complete, buffers are swapped
+//
+// This ensures that processing order doesn't affect results - all systems
+// see the same input state regardless of execution order.
+
+use crate::ecs::{Component, ComponentMeta, meta_of};
+
+/// A single component column (raw byte storage with double-buffering).
+/// 
+/// Uses two buffers for deterministic parallel updates:
+/// - Buffer 0 and Buffer 1
+/// - One is "current" (read), one is "next" (write)
```

Comment:
Nit: you called them "Buffer 0" and "Buffer 1" above, so saying "One is..." makes it sound like you're referring to Buffer 1. Please rename to something like "Buffer A and Buffer B" or "Blue Buffer and Green Buffer" to avoid confusion.